packages/wonder-blocks-modal/components/modal-panel.js

```diff
@@ -176,18 +181,14 @@ const styleSheets = {
             color: Color.white,
         },
 
-        hasTitleBar: {
-            paddingTop: 32,
-        },
-
         hasFooter: {
             paddingBottom: 32,
         },
     }),
 
     small: StyleSheet.create({
         closeButton: {
-            left: 16,
+            right: 16,
             top: 16,
         },
```

Comment:
I thought that we merged `small` styles will `all` styles... won't this result in the close button having `position: "absolute"` regardless of the size?